tests/utils/dmctl.go

```diff
@@ -58,3 +58,22 @@ func StartTask(ctx context.Context, cli pb.MasterClient, configFile string, work
 
 	return nil
 }
+
```

Comment:
why not moving it to `dm_tools` directory?

Reply:
we put binary entry in `dm_tools` directory, and `utils` directory behave more like pkg